backend/modules/simulation/core/orchestrator.py

```
@@ -0,0 +1,699 @@
+        
+        if not persona_id:
+            return None
+        
+        try:
+            db = SessionLocal()
+            try:
+                persona = db.query(SimulationPersona).filter(SimulationPersona.id == persona_id).first()
+                return persona
+            finally:
+                db.close()
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)